Lombiq.NodeJs.Extensions/Docs/Setup-Windows.md

```diff
@@ -0,0 +1,36 @@
+# Recommended Setup of Node.js on Windows
+
+We use [NVM for Windows](https://github.com/coreybutler/nvm-windows) to easily install, update and manage Node.js on our machines. As an added bonus, it allows us to switch between Node.js versions in a matter of seconds. If you’re on Linux, please follow [this guide](Setup-Linux.md).
+
+If you're not part of Lombiq, you might want to skip the usage of NVM for Windows. In that case, install Node.js via its [installer](https://nodejs.org/en/download/) (we recommend the latest LTS version) and skip down to [Additional Configuration](#additional-configuration).
```

Comment:
If this is for COLI, then instead link to the below section from the wiki page.

Reply:
It's not - it's explicitly for externals.